core-java-modules/core-java-collections-list-7/src/test/java/com/baeldung/sortlistofpair/SortListOfPairUnitTest.java

```diff
@@ -0,0 +1,78 @@
+package com.baeldung.sortlistofpair;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Comparator;
+import java.util.List;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.Lists;
+
+public class SortListOfPairUnitTest {
+
+    private List<Pair<String, Integer>> getUnsortedInput() {
+        return Lists.newArrayList(
```

Comment:
Let's avoid using Guava and do List.of or Arrays.asList

Reply:
ok, I'll refactor it using Arrays.asList()